openacademy/wizard.py

```diff
@@ -0,0 +1,19 @@
+from odoo import models, fields, api
+
+
+class Wizard(models.TransientModel):
+    _name = 'openacademy.wizard'
+    _description = 'Wizard to schedule sessions'
+
+    def _default_session(self):
+        return self.env['openacademy.session'].browse(self._context.get('active_ids'))
+
+    session_ids = fields.Many2many(
+        'openacademy.session',
+        string="Session", required=True, default=_default_session)
+    attendee_ids = fields.Many2many('res.partner', string="Attendees")
```

Comment:
Fields before methods please

```suggestion
     session_ids = fields.Many2many(
        'openacademy.session',
        string="Session", required=True, default=_default_session)
    attendee_ids = fields.Many2many('res.partner', string="Attendees")
    
    def _default_session(self):
        return self.env['openacademy.session'].browse(self._context.get('active_ids'))

    
```

Reply:
In this case, the method is used to retrieve tbe default session and then populate the session fields in the wizard, so I guess it makes sense to leave the method first. Is there any other way to do it?

Reply:
It doesn't matter if the method is used on the field, as long as it exists it will be executed.

You can see the [Models](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#37models) section in the guidelines it talks about the order of methods, fields and attributes..

Reply:
I tried to place the field before the method but it throws when I do -u openacademy:

NameError: name '_default_session' is not defined

Reply:
Try this 
session_ids = fields.Many2many('openacademy.session',string="Session", required=True, **default=lambda self: self._default_session()**)

Please see the example in the guidelines

Reply:
using the lambda function worked